test/e2e/common/configmap.go

```diff
@@ -174,6 +176,20 @@ var _ = ginkgo.Describe("[sig-node] ConfigMap", func() {
 		}, metav1.CreateOptions{})
 		framework.ExpectNoError(err, "failed to create ConfigMap")
 
+		ginkgo.By("setting a watch for the ConfigMap")
+		// setup a watch for the ConfigMap
+		resourceWatchTimeoutSeconds := int64(60)
```

Comment:
Magic number, explain why you are choosing this, preferably extract into a constant up top

Reply:
Adjusted in 387c686bfe1